string regex
---
string topicNameNewTopic
string topicNameTopicRemoved
string[] existingTopics